lib/CustomFieldsFilters/utils.test.js

```diff
@@ -0,0 +1,42 @@
+import { CUSTOM_FIELDS } from '../../test/jest/fixtures/customFields';
+import { buildArrayFieldQuery, buildDateRangeQuery } from '../AcqList';
```

Comment:
Could you kindly split the named imports into multiple rows if there is more than one?